Modules/Subscriptions/Tests/Feature/SubscriptionTest.php

```
@@ -0,0 +1,220 @@
+use Modules\Subscriptions\Models\Subscription;
+use Modules\Subscriptions\Services\SubscriptionService;
+use PHPUnit\Framework\Attributes\Group;
+use PHPUnit\Framework\Attributes\Test;
+
+class SubscriptionTest extends AbstractCompanyPanelTestCase
+{
+    #[Test]
+    #[Group('smoke')]
+    public function it_lists_subscriptions(): void
+    {
+        $subscription = Subscription::factory()
```

PHPMD (2.15.0)

[warning] Avoid unused local variables such as '$subscription'. (undefined)

(UnusedLocalVariable)